Network/AMQP/Internal.hs

```diff
@@ -486,6 +486,14 @@ data Channel = Channel {
                     chanExceptionHandlers :: MVar [CE.SomeException -> IO ()]
                 }
 
+data ChanThreadKilledException = ChanThreadKilledException { cause :: CE.SomeException }
```

Comment:
That is true, otherwise there is no way to recognize this exception inside the callback. Should I expose it in `AMQP.Types` then?